### DISCUSSION TITLE
Feature request: add env var for vpn server

### DISCUSSION THREAD

**Discussion by himred** [haugene/docker-transmission-openvpn#2402] [Ideas]
In some scenario, the scripts can select a VPN server by itself if we configure them to do so.
In that case it could be super useful to have somewhere (env var, file, ....) the name and/or IP of the selected server.
Thus in our custom scripts (post/pre start) we can have that information if we need to do something with it.

**Comment by haugene** [comment]
I think you should already have this. It depends on where in the startup process you are of course, the server needs to have been chosen. But it is set as env var CONFIG and should be available in `/scripts/routes-post-start.sh` as well as transmission pre/post start and stop scripts.

I just tested mounting a script:
```bash
echo "HERE IT IS"
echo $CONFIG
```

and got the expected logs
>openvpn  | HERE IT IS
openvpn  | /etc/openvpn/pia/de_berlin.ovpn

**Reply by himred** [reply]
Indeed ! I missed that in the documentation, thank you so much
